2009-07-19  Michael Tautschnig  <mt@debian.org>

	* setup-storage: New version number for experimental.
Index: trunk/bin/setup-storage
===================================================================
--- trunk.orig/bin/setup-storage
+++ trunk/bin/setup-storage	
@@ -52,7 +52,7 @@
 
 package FAI;
 
-my $version = "1.1";
+my $version = "1.1+exp";
 
 # command line parameter handling
 use Getopt::Std;
